Clue/Classes/Writers/JSONWriter.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  JSONWriter.swift
+//  Clue
+//
+//  Created by Prearo, Andrea on 4/25/17.
+//  Copyright © 2017 Ahmed Sulaiman. All rights reserved.
+//
+
+import Foundation
+
+public class JSONWriter: NSObject {
```

Comment:
Makes sense. I can create a new PR to migrate `CLUDataWriter` to Swift (as `DataWriter`) and use it as the base class for `JSONWriter` (as you suggested). Does this work for you?